packages/card-react/src/CardAction.tsx

```diff
@@ -0,0 +1,9 @@
+import { MouseEventHandler } from "react";
```

Comment:
burde det ikke være ClickEventHandler for å få med touch?

Reply:
Har du eksempler på hvor ClickEventHandler har blitt brukt andre steder?